modules/ROOT/partials/integrations/laravel-quick-start.adoc

```diff
@@ -1,13 +1,17 @@
-{% if productSource == "cloud" %}
-  :scriptSource: site.cdnurl
-  :packageSource: site.cloudname
-{% elsif productSource == "composer" %}
-  {% assign scriptSource = "{{ asset('js/tinymce/tinymce.min.js') }}" %}
-  :packageSource: https://packagist.org/packages/tinymce/tinymce[TinyMCE Composer package]
-{% else %}
-  {% assign scriptSource = "{{ asset('js/tinymce/tinymce.min.js') }}" %}
-  :packageSource: TinyMCE .zip package
-{% endif %}
+ifeval::["{productSource}" == "cloud"]
+:scriptSource: {cdnurl}
+:packageSource: {cloudname}
+endif::[]
+ifeval::["{productSource}" == "composer"]
+:scriptSource: "{ asset('js/tinymce/tinymce.min.js') }"
+:packageSource: https://packagist.org/packages/tinymce/tinymce[TinyMCE Composer package]
+endif::[]
+ifeval::["{productSource}" != "cloud"]
+ifeval::["{productSource}" != "composer"]
+:scriptSource: "{ asset('js/tinymce/tinymce.min.js') }"
+:packageSource: TinyMCE .zip package
+endif::[]
+endif::[]
```

Comment:
```suggestion
ifeval::["{productSource}" == "zip"]
:scriptSource: "{ asset('js/tinymce/tinymce.min.js') }"
:packageSource: TinyMCE .zip package
endif::[]
```
While not 100% the same as the old liquid version, these are the 3 states we have so it's a bit cleaner to do this instead. You'll also see it's what was done below.

Reply:
yes, I was not sure that we only have 3 states, I'll change it

Reply:
done